sound/soc/codecs/rt711-sdca.h

```diff
@@ -165,9 +165,9 @@ struct sdw_stream_data {
 #define RT711_BUF_ADDR_HID2			0x44030020
 
 /* RT711 SDCA function topology */
-#define FUN_JACK_CODEC 0x01
-#define FUN_MIC_ARRAY 0x02
-#define FUN_HID 0x03
+#define FUNCTION_JACK_CODEC 0x01
+#define FUNCTION_MIC_ARRAY 0x02
+#define FUNCTION_HID 0x03
```

Comment:
I think we are confusing things here.

This is not the topology, that's defined by the function type.
It looks like the Function number, which is device specific and used for addressing?

Reply:
> This is not the topology, that's defined by the function type.
> It looks like the Function number, which is device specific and used for addressing?

Yes. This function number was defined by Realtek and used for the hierarchical addressing.
This function number should not be the same as the function type.

Reply:
I changed the definitions to 'FUNC_NUM_' for avoiding confusion.